cmake-ide.el

```diff
@@ -441,6 +441,9 @@ the object file's name just above."
 (defun cmake-ide--get-build-dir ()
   "Return the directory name to run CMake in."
   (when (not cmake-ide-dir) (setq cmake-ide-dir (make-temp-file "cmake" t)))
+  (when (not (or (string-prefix-p "/" cmake-ide-dir)
+				 (string-prefix-p "~" cmake-ide-dir)))
+	(setq cmake-ide-dir (concat (cmake-ide--locate-cmakelists) cmake-ide-dir)))
```

Comment:
Well, I have to admit that I am new to just about everything involved in this and only tested it on my system so apologies for anything looking like written by a mad man.
I included tilde because that is "kind of" absolute under linux where tilde is the current users home directory.
And looking at the <code>expand-file-name</code> function it looks like I reinvented the wheel a little.
I shall replace the weird stuff with a sane(r) implementation based on that function soon :-)